.github/workflows/update_ui.yml

```diff
@@ -0,0 +1,65 @@
+name: Update UI
+
+on:
+  workflow_dispatch:
+    inputs:
+      uninstall:
+        description: UI versions to uninstall (space delimited)
+        required: false
+      install:
+        description: UI versions to install (space delimited)
+        required: false
+
+jobs:
+  ui:
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    env:
+      BRANCH: update-ui-${{github.run_number}}
+
+    steps:
+      - name: checkout cylc-doc
+        uses: actions/checkout@v2
+
+      - name: Configure git
+        uses: cylc/release-actions/configure-git@v1
+
+      - name: Create Branch
+        run: |
+          git checkout -b "${BRANCH}"
+
+      - name: uninstall
+        run: |
+          for version in ${{ github.event.inputs.uninstall }}; do
+            git rm -rf "cylc/uiserver/ui/${version}"
+            git commit -m "ui: uninstall ${version}"
+          done
+
+      - name: install
+        env:
+          RELEASES: https://github.com/cylc/cylc-ui/releases/download
+        run: |
+          for version in ${{ github.event.inputs.install }}; do
+            wget \
+              "$RELEASES/${version}/cylc-ui-${version}-dist.zip" \
+              -O "${version}.zip"
+            mkdir -p "cylc/uiserver/ui/${version}/"
+            unzip 0.2.zip -d "cylc/uiserver/ui/${version}/"
```

Comment:
Shouldn't 0.2 zip be something like $version.zip?

Reply:
🤦 well you can tell which version I tested it with!